scaleway/helpers.go

```diff
@@ -21,6 +21,34 @@ import (
 	"golang.org/x/xerrors"
 )
 
+// RegionalID represents an ID that is linked with a region, eg fr-par/11111111-1111-1111-1111-111111111111
+type RegionalID struct {
+	ID     string
+	Region scw.Region
+}
+
+func expandRegionalID(id interface{}) RegionalID {
+	region, ID, _ := parseRegionalID(id.(string))
```

Comment:
The idea Is to reverse dependence `expandRegionalID` should be the correct function to use.
`parseRegionalID` should rely on `expandRegionalID`